playwright/cdp.js

```diff
@@ -1,17 +1,36 @@
+import fs from 'fs';
+
 // Import the Chromium browser into our scraper.
 import { chromium } from 'playwright';
 
-// options passed to the browser.
-let browser_options = {};
-
-// chrome browser path
-if (process.env.CHROME_PATH) {
-    browser_options.executablePath = process.env.CHROME_PATH;
+// check if browser path if a local path or an URL
+let browserPath = process.env.BROWSER_PATH;
```

Comment:
IMO if we decide to use the CDP url, we should accept only URL version.
So I think we have to:
* remove the the option to start chrome process here
* add a clear command line to start the chrome in the README